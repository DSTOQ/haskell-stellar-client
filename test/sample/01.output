TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            PublicKeyEd25519
              { _publicKeyEd25519 =
                  64515967999955780713154879327851495591106351114761682443903344459951597142437
              }
        , _fee = Fee { _fee = 100 }
        , _seqNum = SequenceNumber { _sequenceNumber = 43876020096336346 }
        , _timeBounds = Nothing
        , _memo =
            MemoHash
              Hash
                { _hash =
                    "05aa5cdf46128d0cfa6af5b609a1e20174e9223b94394a293d6060735535511d"
                }
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    Payment
                      PaymentOp
                        { _destination =
                            PublicKeyEd25519
                              { _publicKeyEd25519 =
                                  23599169990946464707245581489179011315953449909347828706960182488016750576992
                              }
                        , _asset = AssetNative
                        , _amount = 20000000
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint "00e7c1a5"
          , _signature =
              Signature
                "7734798f246725cbc27ecca0775f621ccd2fb1d54202f1a1640a29dcc60d7df75f1b502e32296b43f1172763d15a50e635f4a18a66361725fcac3fef481d8e00"
          }
      ]
  }