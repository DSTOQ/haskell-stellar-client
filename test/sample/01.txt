AAAAAI6ivm062fbz0Dbjn6c7KNkghzzBCrL5HswgIt8A58GlAAAAZACb4QIAAAHaAAAAAAAAAAMFqlzfRhKNDPpq9bYJoeIBdOkiO5Q5Sik9YGBzVTVRHQAAAAEAAAAAAAAAAQAAAAA0LKgo0LvbHAuU2vN3hXQEsXscIgE8kQl17QfIhm8FYAAAAAAAAAAAATEtAAAAAAAAAAABAOfBpQAAAEB3NHmPJGcly8J+zKB3X2IczS+x1UIC8aFkCincxg19918bUC4yKWtD8RcnY9FaUOY19KGKZjYXJfysP+9IHY4A
