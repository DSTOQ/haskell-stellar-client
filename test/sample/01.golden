TransactionEnvelope
  { _transaction =
      Transaction
        { _sourceAccount =
            GCHKFPTNHLM7N46QG3RZ7JZ3FDMSBBZ4YEFLF6I6ZQQCFXYA47A2KCQI
        , _fee = FeeStroops { _feeStroops = 100 }
        , _seqNum = 43876020096336346
        , _timeBounds = Nothing
        , _memo =
            MemoHash
              05 aa5cdf46128d0cfa6af5b609a1e20174e9223b94394a293d6060735535511d
        , _operations =
            [ Operation
                { _sourceAccount = Nothing
                , _body =
                    Payment
                      PaymentOp
                        { _destination =
                            GA2CZKBI2C55WHALSTNPG54FOQCLC6Y4EIATZEIJOXWQPSEGN4CWAXFT
                        , _asset = AssetNative
                        , _amount = NonNegativeInt64 (Refined 20000000)
                        }
                }
            ]
        }
  , _signatures =
      [ DecoratedSignature
          { _hint = SignatureHint { _signatureHint = "00e7c1a5" }
          , _signature =
              Signature
                { _signature =
                    "7734798f246725cbc27ecca0775f621ccd2fb1d54202f1a1640a29dcc60d7df75f1b502e32296b43f1172763d15a50e635f4a18a66361725fcac3fef481d8e00"
                }
          }
      ]
  }